src/aliases.cpp

```diff
@@ -159,7 +159,7 @@ static void addValidAliasToMap(std::string_view alias)
 
 static std::string escapeAlias(std::string_view value)
 {
-  std::string newValue = substituteStringView(value,"^^","\\ilinebr ");
+  std::string newValue = substituteStringView(value,"^^","@ilinebr ");
```

Comment:
Why we have additional space at the end of "@ilinebr "?

Reply:
- symmetry
- otherwise the command `ilinebr` cannot be correctly determined e.g. when a "normal" word is following

Reply:
It create 4 spaces instead of 3 spaces in my example.

Reply:
maybe it is better to replace `^^ ` (with space) by `@ilinebr ` (with space) and then `^^` (without space) by `@ilinebr ` (with space), so we don't increase the amount of existing whitespace.

Reply:
@doxygen 
Currently the replacement of `^^` is `\ilinebr ` so with space and, as far as I remember, this was done as otherwise the command ilinebr cannot be correctly determined e.g. when a "normal" word is following.

Reply:
I understand. With my suggestion `^^nospace` would still be replaced by `@ilinebr nospace` (adding an extra space) and `^^ onespace` would be replaced by `@linebr onespace` (so no extra space).

Reply:
@doxygen I missed the subtle difference of when already a space is present after the `^^` no extra space is added otherwise one is is added. Might be a good idea (@TruongMinh89 would also mean that your workaround, 2 versus 3 spaces, is not necessary anymore).

Reply:
I've just updated this proposed pull request